tox.ini

```diff
@@ -43,7 +43,9 @@ commands =
 passenv = HOME
 basepython = python3
 changedir = ./examples/
-deps = mechanical-markdown
+deps =
+    mechanical-markdown
+    grpcio-status>=1.60.0
```

Comment:
That's fine - but in the future just use a `requirements.txt` for your specific example you are adding.